controllers/gce/firewalls/firewalls.go

```diff
@@ -60,9 +60,15 @@ func (fr *FirewallRules) Sync(nodePorts []int64, nodeNames []string) error {
 	// instead of the whole name.
 	name := fr.namer.FrName(suffix)
 	rule, _ := fr.cloud.GetFirewall(name)
+
+	firewall, err := fr.createFirewallObject(suffix, "GCE L7 firewall rule", nodePorts, nodeNames)
```

Comment:
We pass in `suffix` though we just computed the name above. Could we either delete line 61 and change uses to "firewall.Name" or pass in the name?